### ISSUE TITLE
[1.2.24] - Boost Update Issue

### ISSUE THREAD

**Issue by eschoeller** [Cacti/cacti#5421]
I'm running a system with 5 remote pollers. One of the pollers experienced a hardware failure. I moved all the devices off that poller onto another remote poller. Once I did that the system became unstable and I haven't been able to recover it. On-demand RRD updates are no longer functioning. So for any data sources being polled by remote pollers, those data sources are only updated after the boost process runs, regardless of when the graphs are viewed. 

I have encountered this problem before when I attempted to move a large number of devices between remote pollers. I can't say for sure how many devices triggers this problem, and I can't for the life of me recall how I've fixed it in the past. I had hoped that over time this "bug" would eventually get resolved but it still appears to be present somewhere in the boost feature. 

It's challenging for me to figure out how to really articulate the steps needed to reproduce this or provide any diagnostic data to help show what's going on in terms of error messages, etc. since I'm not finding anything in the logs that indicate a problem. 
I'm hoping for some guidance on what I should be looking for. It's hard to parse through the debug logs to find anything of substance relating to this problem. I've rebuilt the poller cache, the snmp cache, run full sync's to the remote pollers. I've stopped the pollers entirely, turned off boost, restarted mysql. Any hints would be greatly appreciated!

**Comment by TheWitness** [comment]
How about a zoom Eric?

**Comment by eschoeller** [comment]
I'm certainly up for that! When works best for you? I only have a meeting from 1-1:30 MST today.

**Comment by TheWitness** [comment]
I have to go look at a car @4:00pm EDT.  So, say 5:30EDT?  Send your contact details to thewitness@cacti.net in the mean time just in case theve changed.

**Comment by TheWitness** [comment]
@eschoeller, so what's the current status?

**Comment by eschoeller** [comment]
Sorry I hopped a train to NYC and I'll probably be unavailable the next few days .. 
moved everything to the main poller and that stabilized it for now. I'll try to move a couple test devices back to remote pollers and see if it works ... 
since I've seen this happen a few times now I sorta want to figure out what's causing it. First I need to figure out how to fix/replicate it I guess.

**Comment by TheWitness** [comment]
Any updates @eschoeller ?

**Comment by netniV** [comment]
Are there any further updates on this as we are planning to release?

**Comment by TheWitness** [comment]
@eschoeller, any updates from your side?

**Comment by TheWitness** [comment]
@eschoeller ping!  I'm going to close.  Let me know how it's going through Slack.
